crates/transaction-pool/src/validate/eth.rs

```diff
@@ -807,6 +803,7 @@ pub fn ensure_intrinsic_gas<T: EthPoolTransaction>(
     transaction: &T,
     fork_tracker: &ForkTracker,
 ) -> Result<(), InvalidPoolTransactionError> {
+    use revm_primitives::SpecId;
```

Comment:
how come this use statement is local to this function?

Reply:
I want to limit revm imports to only this function